### DISCUSSION TITLE
Jitsi helm chart compatibility with Jitsi Meet

### DISCUSSION THREAD

**Discussion by anirbandas18** [jitsi-contrib/jitsi-helm#82] [Q&A] [answered]
Hi,

I can see that each release of jitsi helm chart has a helm chart version and an associated Jitsi Meet version with it. 

Is there any compatibility matrix that encapsulates which version of Jitsi Meet is compatible with which version of Jitsi Helm chart?

**Comment by spijet** [comment]
Hello @anirbandas18!

I usually try to mention the breaking changes in chart release notes. Most of the time these changes correspond to specific Jitsi Meet image releases which introduce something new or different under the hood, like UDP-only JVB operation or automatic DNS detection for k8s environments.

**Comment by anirbandas18** [comment]
Where can I find these chart release notes?

**Reply by spijet** [reply, marked as answer]
You can find them on the [Releases](https://github.com/jitsi-contrib/jitsi-helm/releases) page.
